integrations/amazon_bedrock/tests/test_generator.py

```diff
@@ -294,17 +300,64 @@ def test_long_prompt_is_not_truncated_when_truncate_false(mock_boto3_session):
         ("eu.mistral.mixtral-8x7b-instruct-v0:1", MistralAdapter),  # cross-region inference
         ("us.mistral.mistral-large-2402-v1:0", MistralAdapter),  # cross-region inference
         ("mistral.mistral-medium-v8:0", MistralAdapter),  # artificial
-        ("unknown_model", None),
     ],
 )
 def test_get_model_adapter(model: str, expected_model_adapter: Optional[Type[BedrockModelAdapter]]):
     """
     Test that the correct model adapter is returned for a given model
     """
-    model_adapter = AmazonBedrockGenerator.get_model_adapter(model=model)
+    model_adapter = AmazonBedrockGenerator.get_model_adapter(model=model, model_family=None)
```

Comment:
If model_family is set to None by default, we don't need to pass it here explicitly.